homework-submission/week4-class16/src/stores/TodosStore.jsx

```diff
@@ -0,0 +1,90 @@
+import { observable, computed, action } from "mobx";
+import Data from "../Data.json";
+import { configure } from "mobx";
+configure({ enforceActions: true });
+
+class TodosStore {
+  @observable
+  edit = false;
+  @observable
+  Data = Data;
+
+  @action
+  handleCheck = todoId => {
+    const newData = this.Data.map(todo => {
+      if (todo.id === todoId) {
+        return { ...todo, done: !todo.done };
+      }
+      return todo;
+    });
+    this.Data = newData;
+  };
+  @action
+  handleDelete = todoId => {
+    const filteredTodos = this.Data.filter(todo => {
+      return todo.id !== todoId;
```

Comment:
as I told before this is okay in MobX, while in the state this shouldn't be the case.